prac_08/taxi_simulator.py

```diff
@@ -17,20 +17,30 @@ def main():
             user_input = input("Choose taxi: ").lower()
             current_taxi = taxis[int(user_input)]
```

Comment:
You may need to add some error checking with user inputs. they will crash if the user enters a string by mistake and your code try's to convert it into a integer.

Reply:
I didn't add error checking here because it wasn't explicitly asked for, but I do like my error checking so...